### PR TITLE
Document cmake and xz requirements

### PR DESCRIPTION
**Pull request by tamird**

### CODE REVIEW

**Comment by benesch** [comment]
:lgtm:

---

Reviewed 1 of 1 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/cockroach/15060#-:-Ki1zQ_TR1sVDbfLfxi2:bnfp4nl)*

**Comment by benesch** [comment]
Reviewed 1 of 1 files at r2.
Review status: all files reviewed at latest revision, all discussions resolved, some commit checks pending.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/cockroach/15060)*

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -9,7 +9,10 @@
     the Makefile enforces the specific version required, as it is updated
     frequently.
   - Git 1.8+
-  - Bash
+  - Bash (4+ is preferred)
+  - Make (3.81+ is known to work)
```
Comment (bdarnell) [inline CONTRIBUTING.md]: Specify "GNU Make"

Comment (tamird) [inline CONTRIBUTING.md]: Done.

**Pull request merged**